content/applications/sales/members/members_analysis.rst

```diff
@@ -0,0 +1,54 @@
+================
+Members analysis
+================
+
+.. |caret| replace:: :icon:`fa-caret-down` :guilabel:`(down)` icon
+.. |pivot| replace:: :icon:`oi-view-pivot` :guilabel:`(pivot)` icon
+.. |list| replace:: :icon:`oi-view-list` :guilabel:`(list)` icon
+
+The *Members Analysis* report provides a detailed overview of all memberships sold and managed
+within the *Members* application.
+
+To access the *Memberships Analysis* report, navigate to :menuselection:`Members app --> Reporting`.
+
+Navigate the report
+===================
+
+The default *Members Analysis* report displays a bar chart, measuring the :guilabel:`Quantity` of
+memberships against the amount of :guilabel:`Revenue Done`. The data is grouped by the assigned
+salesperson, then by the month the membership begins.
+
+.. image:: members_analysis/example-report.png
+   :align: center
+   :alt: A sample of the members analysis report.
+
+Additional :ref:`Group By <search/group>` options include the following:
+
+- :guilabel:`Salesperson`: the internal user in charge of the contact. This information is
+  managed in the :guilabel:`Salesperson` field on the :guilabel:`Sales & Purchase` tab on the
```

Comment:
```suggestion
  managed in the *Salesperson* field on the *Sales & Purchase* tab on the
```
opting for italics here because this field and tab aren't being presently shown on the screen, but would otherwise be granted a GUI label